agent/consul/state/acl.go

```diff
@@ -978,6 +1009,10 @@ func (s *Store) ACLTokenBatchGet(ws memdb.WatchSet, accessors []string) (uint64,
 }
 
 func (s *Store) aclTokenGetTxn(tx *memdb.Txn, ws memdb.WatchSet, value, index string) (*structs.ACLToken, error) {
+	return s.aclTokenGetTxnFixupLinks(tx, ws, value, index, true)
+}
+
+func (s *Store) aclTokenGetTxnFixupLinks(tx *memdb.Txn, ws memdb.WatchSet, value, index string, fixLinks bool) (*structs.ACLToken, error) {
```

Comment:
Do we still need both variants? I think you said that we don't actually need to avoid the fixup any more right?